libs/cypress-indexeddb/src/lib/object-store-metadata.ts

```diff
@@ -32,6 +42,16 @@ export function keys(store: IDBObjectStore): Promise<IDBValidKey[]> {
     });
 }
 
+/**
+ * List all entries in store
+ *
+ * @remarks The `entries` method lists all the entried available in the provided store.
+ *
+ * @param store `IDBObjectStore` instance
+ *
+ * @returns T[]
+ * @throws {Error} If the connections fails to open or the read operation fails.
```

Comment:
technically it only throws an error if it is chained off from a method that does not return an object store.